pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -123,7 +132,8 @@ func ipSecReplacePolicyOut(src, dst *net.IPNet) error {
 		Value: linux_defaults.RouteMarkEncrypt,
 		Mask:  linux_defaults.RouteMarkMask,
 	}
-	ipSecAttachPolicyTempl(policy, getIPSecKeys(dst.IP), src.IP, dst.IP)
+	key := getIPSecKeys(dst.IP)
+	ipSecAttachPolicyTempl(policy, key, src.IP, dst.IP)
```

Comment:
yeah some errors are actually OK here, such as EEXIST if we get duplicate node entries. We should probably be a bit more robust and test for eexist otherwise error out.

Reply:
Yeah an EEXIST error is not a 'hard" error to abort on. To be more robust we should check for EEXIST and only error if its some other type.

Reply:
actually I misread the comment this was just an omission.